### DISCUSSION TITLE
[Unstable] Some issues about MUX/DEMUX components

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#14345] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

move_output pin of DEMUX component are not function inside circuit box
move_output and set_output(is this supposed to be move_input and set_input??) pin of MUX component are complete not function, whatever inside circuit box or no

### Reproduction steps

just test there pins

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested, fix confirmed to be working correctly. Coming in the next patch. Closing discussion.
